src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.ts

```diff
@@ -80,9 +83,21 @@ export class RgwBucketListComponent {
       canBePrimary: (selection: CdTableSelection) => selection.hasMultiSelection
     };
     this.tableActions = [addAction, editAction, deleteAction];
+    this.timeConditionReached();
+  }
+  timeConditionReached() {
+    clearTimeout(this.staleTimeout);
+    this.ngZone.runOutsideAngular(() => {
+      this.staleTimeout = window.setTimeout(() => {
+        this.ngZone.run(() => {
+          this.isStale = true;
+        });
+      }, 10000);
+    });
   }
 
   getBucketList(context: CdTableFetchDataContext) {
+    this.timeConditionReached();
```

Comment:
You are missing setting `isStale` to `false` when calling `timeConditionReached()`.
You can either put the code here:
```suggestion
    this.isStale = true;
    this.timeConditionReached();
```
or add it in the 1st line of `timeConditionReached()`

Reply:
I guess, this.isStale=false, right?